fuse/eval/metrics/classification/metrics_ensembling_common.py

```diff
@@ -0,0 +1,41 @@
+from typing import Optional, Sequence, Hashable
+from fuse.eval.metrics.libs.ensembling import Ensembling
+from fuse.eval.metrics.utils import PerSampleData
+
+import pandas as pd
+
+import numpy as np
+
+from fuse.eval.metrics.metrics_common import MetricDefault
+from functools import partial
+
+
+class MetricEnsemble(MetricDefault):
```

Comment:
not sure, it can also be used for regression or otherwise. 
maybe it shouldn't be in the classification dir if so, but I wasn't sure where to put it
it is separate from metrics_classification_common.py file